sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -1137,37 +1137,24 @@ def model_group_by_key(contents, output_path):
      | beam.io.WriteToText(output_path))
 
 
-def model_co_group_by_key_tuple(email_list, phone_list, output_path):
+def model_co_group_by_key_tuple(p, emails, phones, output_path):
   """Applying a CoGroupByKey Transform to a tuple."""
   import apache_beam as beam
-  with TestPipeline() as p:  # Use TestPipeline for testing.
-    # [START model_group_by_key_cogroupbykey_tuple]
-    # Each data set is represented by key-value pairs in separate PCollections.
```

Comment:
Are you dropping the information in this comment, should it go to the test file?

Reply:
I think it's already pretty self-explanatory in the test file since we can see the actual lists used to create the PCollections.